src/Controls/src/Core/SolidColorBrush.cs

```diff
@@ -53,7 +53,7 @@ public override bool Equals(object obj)
 		/// <include file="../../docs/Microsoft.Maui.Controls/SolidColorBrush.xml" path="//Member[@MemberName='GetHashCode']/Docs/*" />
 		public override int GetHashCode()
 		{
-			return -1234567890 + Color.GetHashCode();
+			return -1234567890 + (Color?.GetHashCode() ?? 0);
```

Comment:
```suggestion
			return HashCode.Combine(Color);
```

HashCode.Combine ensure your implementation is right, and I don't complain

Reply:
while you're at it, please fix ImageBrush and GradientStop using the same bad seed

Reply:
@StephaneDelcroix It seems HashCode is not supported in .NET Standard 2.0, and the build fails when trying to compile the solution.
<img width="563" alt="image" src="https://github.com/user-attachments/assets/8d08f19d-9d77-4749-a103-7d2e5b5c4cb6" />